src/pybind/mgr/cephadm/services/cephadmservice.py

```diff
@@ -1030,6 +1030,37 @@ def prepare_create(self, daemon_spec: CephadmDaemonDeploySpec) -> CephadmDaemonD
             'value': frontend
         })
 
+        if spec.rgw_user_counters_cache:
+            ret, out, err = self.mgr.check_mon_command({
+                'prefix': 'config set',
+                'who': utils.name_to_config_section(daemon_spec.name()),
+                'name': 'rgw_user_counters_cache',
+                'value': str(spec.rgw_user_counters_cache).lower(),
```

Comment:
can this ever be anything other than "true" if we get here? This variable is an optional bool, so as strings it could be "true", "false", or "none", but of those only True would pass the `if` check. If that is the case, I think we ought to just replace `str(spec.rgw_user_counters_cache).lower()` with `"true"`. Same would go for the bucket version.